prowler/lib/check/models.py

```diff
@@ -573,6 +573,37 @@ def __init__(
         self.location = resource_location
 
 
+@dataclass
+class CheckReportNHN(Check_Report):
```

Comment:
```suggestion
class Check_Report_NHN(Check_Report):
```

Just to follow all the names format.